Dockerfile.tests

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:18.04 AS build
-RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev
+FROM ubuntu:20.04 AS build
+RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev libslirp-dev
```

Comment:
Can we roll back to ubuntu:18.04 and build libslirp from git?
Because
* Ubuntu 20.04 isn't GA yet
* Upgrading to 20.04 seems to have adopted new `clang-format` style
* Tests should be executable with arbitrary `ARG LIBSLIRP_COMMIT` (I can open follow-up PR for this)

Reply:
I dropped 20.04 bump. But make indent still failiing. Any idea?

Reply:
doesn't seem dropped

Reply:
`ubuntu:20.04` in `Dockerfile.buildtests` seems rolled back to 18.04, but you still have `ubuntu:20.04` in `Dockerfile.tests`

Reply:
oops, my bad. thanks!